packages/components/video/UploadVideoModal.tsx

```diff
@@ -146,9 +161,11 @@ export const UploadVideoModal: FC<{
       userIPFSKey || (await generateIpfsKey(selectedNetwork?.id || "", userId));
     if (!pinataJWTKey) {
       console.error("upload file err : No Pinata JWT");
-      setToastError({
+      setToast({
```

Comment:
not in the scope of this PR, just an idea to remind when we have time: I see that setToast has changed the signature and feel that more cumbersome to call. My be we can wrap it in our hook like: successToast(title, msg, type, mode='norman')